hal/src/main/native/sim/MockData/SPIData.cpp

```diff
@@ -13,6 +13,21 @@
 
 using namespace hal;
 
+void InvokeCallback(
+    std::shared_ptr<SpiAutoReceiveDataListenerVector> currentVector,
+    const char* name, uint8_t* buffer, int32_t numToRead,
+    int32_t* outputCount) {
+  // Return if no callbacks are assigned
+  if (currentVector == nullptr) return;
+  // Get a copy of the shared_ptr, then iterate and callback listeners
+  auto newCallbacks = currentVector;
+  for (size_t i = 0; i < newCallbacks->size(); ++i) {
+    if (!(*newCallbacks)[i]) continue;  // removed
```

Comment:
What was "removed"?

Reply:
I think it indicates the callback is null, therefore was removed.  Updated the comment to reflect that

Its worth noting this is very similar to the other `InvokeCallback` functions elsewhere that have the same comment